packages/rs-drive/Cargo.toml

```diff
@@ -79,7 +82,7 @@ default = ["full"]
 fixtures-and-mocks = ["mockall"]
 full = [
     "fixtures-and-mocks",
-    "grovedb/full",
+    "grovedb/estimated_costs",
```

Comment:
Why this change?

Reply:
Yeah, you should also look at @QuantumExplorer pr: https://github.com/dashpay/platform/pull/1340